bin/_test-helpers.sh

```diff
@@ -120,6 +126,12 @@ handle_tests_input() {
     tests_usage "$0" >&2
     exit 64
   fi
+
+  if [ -z "$test_name" ] && [ -n "$skip_cluster_delete" ]; then
+    echo "Error: must provide --name when using --skip-cluster-delete" >&2
+    tests_usage "$0" >&2
+    exit 64
+  fi
```

Comment:
Good call

Reply:
Actually do we want to enforce this? We don't for `skip_cluster_create`, and we don't pass `--name` in when using it for ARM64 tests. Could be nice to have the whole cluster stick around if running all tests.

Reply:
When all the tests are run, the beginning of each test expects a clean slate otherwise it'll fail. I could make it so `--skip-cluster-delete` does clean the cluster after each test except for the last one, if you think that'd be valuable.

Reply:
Nope I think this is better then!